polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -1057,41 +1135,114 @@ mod tests {
 
 		assert_eq!(new_ccr.descriptor.version(), CandidateDescriptorVersion::Unknown);
 		assert_eq!(
-			new_ccr.check_core_index(&BTreeMap::new()),
+			new_ccr.check_ump_signals(&BTreeMap::new()),
 			Err(CommittedCandidateReceiptError::UnknownVersion(InternalVersion(100)))
-		)
+		);
 	}
 
 	#[test]
-	fn test_ump_commitment() {
+	fn test_ump_commitments() {
```

Comment:
This is testing happy paths of `CommittedCandidateReceiptV2::check_ump_signals`.

Reply:
Particularly: 
- only Selected core
- only approved peer
- both signals - in any order